components/ChoreWeight.tsx

```diff
@@ -48,15 +57,19 @@ export default function ChoreWeight({
           <Card.Actions>
             <View style={s.content}>
               <View style={s.textContainer}>
-                <Text style={{ fontWeight: 'bold', fontSize: 18 }}>
+                <Text
+                  style={{ fontWeight: 'bold', fontSize: 18, color: textColor }}
+                >
                   Value:{' '}
                 </Text>
-                <Text>How energy-demanding is the task?</Text>
+                <Text style={{ color: textColor }}>
+                  How energy-demanding is the task?
+                </Text>
               </View>
               <View>
                 <Badge
                   size={24}
-                  style={[s.badge, { backgroundColor }]}
+                  style={[s.badge]}
```

Comment:
```suggestion
                  style={[s.badge, { color: colors.inverseOnSurface }]}
```
This suggestion isn't strictly necessary, as it works regardless, but it makes sure the color is clearer (especially in dark mode where it's a clearer black than it would be otherwise)

Reply:
Also please note that if you add this here, you should do the same for the badge in ChoreFrequency for consistency